docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -137,9 +139,33 @@ View Alluxio WebUI at `http://master_hostname:19999` and you can see the directo
 
 Alternatively, you can follow the [instructions](Running-Hive-with-Alluxio.html#create-new-tables-from-files-in-alluxio) to create the tables from existing files in Alluxio.
 
-Next, using a single query:
+Next, start your Hive metastore service. Hive metastore listens on port 9083 by default.
+
+```bash
+$ /<PATH_TO_HIVE>/bin/hive --service metastore
+```
+
+The following is an example of the Presto configuration `/<PATH_TO_PRESTO>/etc/catalog/hive.properties` :
+
+```properties
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://localhost:9083
+hive.config.resources=/<PATH_TO_HADOOP>/etc/hadoop/core-site.xml,/<PATH_TO_HADOOP>/etc/hadoop/hdfs-site.xml
 ```
-/home/path/presto/presto-cli-0.191-executable.jar --server masterIp:prestoPort --execute "use default;select * from u_user limit 10;" --user username --debug
+
+Start your Presto server. Presto server runs on port 8080 by default:
+
+```bash
+$ /<PATH_TO_PRESTO>/bin/launcher run
+```
+
+Follow [Presto CLI guidence](https://prestodb.io/docs/current/installation/cli.html) to download to `presto-cli-<PRESTO_VERSION>-executable.jar`,
+rename it to presto, make it executable with `chmod +x`.
```

Comment:
Please use backticks with `presto`.

Does this file have any relationship with `/<PATH_TO_PRESTO>/bin/`?

Renaming a `.jar` file to an executable just works?